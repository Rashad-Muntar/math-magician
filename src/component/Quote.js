import React from 'react';
import '../App.css';

const Quote = () => (
  <div className="quoteWrapper">
    <h1 className>Quote of the day</h1>
    <p>
      Mathematics is not about numbers, equations, computations, or algorithms:
      it is about understanding.
    </p>
    <p>
      â€“William Paul ThurstonHome
    </p>
  </div>
);

export default Quote;
