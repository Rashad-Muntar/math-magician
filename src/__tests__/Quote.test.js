import React from 'react';
import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import Quote from '../component/Quote';

describe('Quote', () => {

  it('Renders Quote component', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
