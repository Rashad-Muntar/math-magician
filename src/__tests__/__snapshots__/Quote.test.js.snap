// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote Renders Quote component 1`] = `
<div
  className="quoteWrapper"
>
  <h1
    className={true}
  >
    Quote of the day
  </h1>
  <p>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding.
  </p>
  <p>
    –William Paul ThurstonHome
  </p>
</div>
`;

exports[`Quote Renders calculator component 1`] = `
<div
  className="quoteWrapper"
>
  <h1
    className={true}
  >
    Quote of the day
  </h1>
  <p>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding.
  </p>
  <p>
    –William Paul ThurstonHome
  </p>
</div>
`;
